packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -888,9 +888,9 @@ class ServiceMap extends ServiceObject implements Map<String, dynamic> {
   @override
   bool containsKey(Object k) => _map.containsKey(k);
   @override
-  void forEach(Function f) => _map.forEach(f);
+  void forEach(void f(String key, dynamic value)) => _map.forEach(f);
```

Comment:
I believe it was simply an error that these two methods did not match the signatures specified by Map<String, dynamic>